### PR TITLE
Neethu

### PR DESCRIPTION
**Pull request by Neethumanu1213**

### CODE REVIEW

File: src/main/java/com/naveenautomation/Pages/MyAccountPage.java [hunk 1]
```diff
@@ -12,15 +12,15 @@ public class MyAccountPage extends Page {
...
 	}
-
 	
 	private static final By phonelink = By.cssSelector("i.fa.fa-phone");
 	private static final By GiftCertificatesLink = By
 			.cssSelector("footer div.row div.col-sm-3:nth-of-type(3) ul li:nth-of-type(2) a");
 	private static final By myAccountText = By.xpath("//h2[text()='My Account']");
 	private static final By changePasswordLink = By.cssSelector("#column-right a:nth-of-type(3)");
-	private static final By passwordChangesSuccessBanner = By.cssSelector("div.alert-success");
+	private static final By SuccessBanner = By.cssSelector("div.alert-success");
```
Comment (PivotITAcademy) [inline src/main/java/com/naveenautomation/Pages/MyAccountPage.java:21]: naming convention is not correct and why you want to change the name

**Review by PivotITAcademy — CHANGES_REQUESTED** [review]
Plz do the changes

File: src/main/java/com/naveenautomation/Pages/MyAccountPage.java (hunk 1 above)
Comment (Neethumanu1213) [reply src/main/java/com/naveenautomation/Pages/MyAccountPage.java]: Based on the review I updated the code. That locator is common for all the banner that is why I changed the name

**Review by PivotITAcademy — APPROVED** [review]
LGTM

**Pull request merged**